### PR TITLE
Add homepage branch

### PR DESCRIPTION
**Pull request by wachirasenior0**
files added; 
- Index.html
- style.css
- test.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse Miniproject</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (brainconnect93) [inline index.html:12]: - [x] I would kindly suggest you stick to the project requirement as much as possible, As stated in the requirement you should have a `.html` file that includes a header with the text "Hello Microverse!". Use the `h1` tag for that. you can achieve using the example below.

Example 👇 
```HTML
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Hello Microverse Miniproject</title>
</head>
<body>
     <h1 class="header">Hello Microverse Project!</h1>
</body>
</html>
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: blue;
+  font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (brainconnect93) [inline style.css:4]: - [x] I would kindly suggest you use the class name given to your `h1` tag in your HTML file to style the content in the CSS file

i.e 👇 

```CSS
.header {
  color: blue;
  font-family: Arial, Helvetica, sans-serif;
}
```

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @wachirasenior0, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive proper Pull Request summary. ✔️  
- Hello Microverse project done ✔️ 
- There is a `.gitignore` file listing `test.md` file and `node_modules` ✔️
- Good use of GitHub flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

- [x] I would kindly suggest you work on the MIT license link as currently the link is throwing error 404, having a working link will make your README file more descriptive and professional.

MIT 👇 
mit

- [x] Secondly, I would kindly suggest you remove all the placeholder text describing what you need to do in each heading of the Readme file. This is just there as a guideline for what you need to do in each heading and it should be removed.

placeholder text like 👇 
> Describe your project in 1 or 2 sentences.
> Describe the tech stack and include only the relevant sections that apply to your project.
> Describe between 1-3 key features of the application.
> Describe how a new developer could make use of your project. e.t.c

- [x] Kindly add the correct tech stack used for this project in the tech stack heading. Currently, I can see `React`, `Express`, and `Postgres` as the tech stack and these are not the tech stack used. Kindly limit your tech stack used to only HTML and CSS for this milestone.
- [x] Kindly add 1-3 pieces of content of what you will like to add in the future to this project in the `future features` heading as currently, the heading is missing the necessary content
- [x] Also in the `Getting Started` heading I would kindly suggest you add all the necessary content to implement to get the project running.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @wachirasenior0, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)


 ## Required Changes ♻️
_Check the comments under the review._

- [x]  I noticed that you had made the [changes requested](https://github.com/wachirasenior0/Hello-Microverse-Miniproject/pull/1#pullrequestreview-1512869944) by the previous reviewer on the `master` branch instead of this branch `add-hompage`, please switch to this feature branch, make those changes and push to GitHub and it will reflect in this pull request. 
- [x] Currently, your README file on the master branch has the getting started section containing only the headers kindly add a proper guide on how users can clone, install, and lunch the project. See the example 👇🏾 

image

This is specific for his project, use this as a guide. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wachirasenior0** [comment]
Pushed the updated version to the add-homepage branch

**Review by shahadat3669 — APPROVED** [review]
Hi  @wachirasenior0 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Good PR title
✅ Good PR description


## Optional suggestions

Great job on your project! 👍 However, there are a couple of points to address:

- [Optional] It's important to follow the GitHub flow properly for your future projects. I recommend using "main" as the base branch name instead of "master". This follows industry best practices and promotes inclusive and respectful language. You can refer to the [GitHub Flow guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) for more information.
- [Optional] I appreciate your efforts towards improving your commits.:+1: It's encouraging to see your commitment to enhancing the commit history. However, there is still room for further improvement. Please try to enhance the clarity and organization of your commit history. :pray:

| Your | Professional |
| --- | --- |
| image | image |

By following these suggestions, you will align with established practices and ensure the smooth progression of your project. Keep up the good work! If you have any further questions, feel free to ask.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**